cmd/rook/ceph/osd_test.go

```diff
@@ -136,7 +136,7 @@ func TestParseDesiredDevices(t *testing.T) {
 	assert.NoError(t, err)
 	devices = string(marshalledDevices)
 
-	result, err = parseDevices(devices)
+	result, _ = parseDevices(devices)
```

Comment:
We should handle the error instead.

`assert.NoError`

Reply:
Done.